Mod2/Lessons/Week1/SQLCRUD.md

```diff
@@ -0,0 +1,132 @@
+
+To check that records have been created, or anytime you want to see the current state of a table, you can run this statement: `SELECT * FROM <tablename>;`
+
+### Practice Insert
+
+> With a partner, use what you know about INSERT INTO to create records in your `orders` table.  You can use [this data](https://docs.google.com/spreadsheets/d/1KX3EhldBiQAVxXoQpQfzveHbhiNerxmmmkMufFe4ZrM/edit#gid=1715070853), or create your own!  Keep in mind that an order should be associated with an existing customer.
+>
+> A note about price: we have set up our database to store price in cents, so if an item costs $45.99, that is 4599 cents.
+> 
+> Reach out to an instructor with any questions or issues that come up!
+
+## Deleting Records
+
+To remove records from our database, we can use the SQL DELETE command.
+
+> With a partner, take a look at the delete statement below and answer the following questions:
+> * What do we need to know about a table in order to delete a record?
+> * What do you think would happen if we tried to delete a record that doesn't exist?
+>
+> Be ready to share out!
+> 
+> ![](/Mod2/Images/Week1/SQL_DELETE.png)
+> 
+> **Warning** The `WHERE` command tells SQL which records to delete.  If you do not use a `WHERE`, the `DELETE` will apply to **all** rows in the table ☠️
+
+When deleting records, we use a `WHERE` command to isolate the single record that we want to remove.  The `WHERE` command will pull all records that match specific criteria; note that in SQL, we make comparisons with a single `=`.
+
+Though deleting records is possible, in practice we do not often remove records from a database. It is better to indicate that a record (maybe a user) is no longer active instead of deleting it.  That way, we can maintain a history of our data.
+
+### Practice DELETE
+
+> With a partner, use what you know about DELETE to remove one or more records in your `orders` table.  If you end up removing all of the records from the orders table (on purpose, or by accident), INSERT some records to be ready for the next sections!
+>
+> Reach out to an instructor with any questions or issues that come up!
+
+## Updating Records
+
+Occasionally, we will need to update records.  We can do that with the `UPDATE` command.
+
+> With a partner, take a look at the update statement below and answer the following questions:
+> * What do we need to know about a table in order to update a record?
+> * What do you think would happen if we tried to update a record that doesn't exist?
+> * What do you think would happen if we did not include a **WHERE** in our statement?
+>
+> Be ready to share out!
+> 
+> ![](/Mod2/Images/Week1/SQL_UPDATE.png)
+
+** Instructor Note ** Be sure to point out that if we forget the `WHERE`, all records in the table will be updated!
```

Comment:
I would encourage instructors to actually type out an update without a where clause, ask for predications from students, and then run it and show how everything changes so folks can really feel what happens.